gcs/src/redux/slices/droneInfoSlice.js

```diff
@@ -28,6 +28,7 @@ const droneInfoSlice = createSlice({
     navControllerData: {
       navBearing: 0.0,
       wpDist: 0.0,
+      loiterRadius: 80.0, // Default loiter radius
```

Comment:
I'll remove these but we don't need to comment when its default as initial state is implied that its default. Only need to add comments to the lines when something different is going on, for example the TODOs later on.